en/services/parameter.md

```diff
@@ -42,19 +51,6 @@ All onboard components should respond to parameter request messages with their I
 > **Tip** *QGroundControl* by default queries all components of a system (it only queries the currently selected system, not all systems) and therefore sends ID 0 (`MAV_COMP_ID_ALL`).
 
 
-## Graphical User Interface in QGroundControl
```

Comment:
@julianoes FYI, I deleted this section because this is all convention - not part of the spec (in fact it is no longer used by QGC - groups are used for sorting).